backend/kernelCI_app/views/hardwareDetailsBootsView.py

```diff
@@ -99,6 +99,15 @@ def _sanitize_records(
 
     # Using post to receive a body request
     @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="hardware_id",
+                description="ID of the hardware",
```

Comment:
```suggestion
                description="ID of the hardware, as the name of the platform/compatible",
```
Please change this on the other hardware endpoints as well